src/main/common/components/ccj/paid-amount-summary.njk

```diff
@@ -35,7 +37,7 @@
             <th scope="row">{{ t('Amount you’re settling for') }}</th>
             <td class="numeric last">
               <span class="font-small">{{ amountSettledFor | numeral }}</span>
-            </td>
+            </td><x></x>
```

Comment:
what is this tag for?

Reply:
must be a result of wrong keyboard shortcut :-)